package.json

```diff
@@ -1,13 +1,15 @@
 {
 	"name": "la1tv-roses-graphics",
-	"version": "1.0.0",
+	"version": "2018.0.1",
```

Comment:
When was the version change from SemVer ?

Reply:
we werent semvering tbf, plus i feel it makes more sense to version on the year and minor/patching

Reply:
Lets not change the versioning scheme in this PR regarding UI live indicators we can switch to this at a later date